instrumentation/opentelemetry/init_test.go

```diff
@@ -44,6 +44,19 @@ func ExampleRegisterService() {
 	defer shutdown()
 }
 
+func TestInitWithCertfileAndSecure(t *testing.T) {
+	cfg := config.Load()
+	cfg.ServiceName = config.String("my_example_svc")
```

Comment:
I might be missing something but I don't think we need all the config values being declared here and that sort of polutes the test. Can we just include those config that are required? otherwise it gives the impression this is required for the test.

Reply:
done